File: SAMPLES/JAVA/test05_while.java
public class test05_while {
    public static void main(String[] args) {
        int i;
        int sum;

        i = 0;
        sum = 0;
        while (i < 10) {
            sum += i;
            i++;
        }
        System.out.println(sum);

        i = 0;
        while (i < 20) {
            if (i == 5) {
                i++;
                continue;
            }
            if (i == 10) {
                break;
            }
            i++;
        }
        System.out.println(i);
    }
}

File: SAMPLES/CSHARP/test05_while.cs
using System;

namespace Test05
{
    class Program
    {
        static void Main()
        {
            int i;
            int sum;

            i   = 0;
            sum = 0;

            while (i < 10)
            {
                sum += i;
                i++;
            }
            Console.WriteLine(sum);

            i = 0;
            while (i < 20)
            {
                if (i == 7)
                {
                    break;
                }
                if ((i % 2) == 0)
                {
                    i++;
                    continue;
                }
                Console.WriteLine(i);
                i++;
            }
        }
    }
}
